rotkehlchen/chain/ethereum/graph.py

```diff
@@ -68,7 +68,7 @@ def query(
             prefix += json.dumps(param_types).replace('"', '').replace('{', '(').replace('}', ')')
             prefix += '{'
 
-        querystr = prefix + querystr
+        querystr += prefix
```

Comment:
this is why the subgraph tests failed. It's the ruff bug, but weird. I did not use autofix so may have done this on my own without paying attention.